.github/workflows/remedy-unexpected-bodies.yml

```
@@ -0,0 +1,144 @@
+
+concurrency:
+  group: remedy-unexpected-bodies
+  cancel-in-progress: false
+
+jobs:
+  remedy:
+    runs-on: ubuntu-latest
+    # `apply` pushes a branch and opens a PR, so it needs more than the read-only default.
+    # audit.yml's own dispatch always passes `apply: true` explicitly, same as a human would.
+    permissions:
+      contents: ${{ inputs.apply && 'write' || 'read' }}
```

actionlint (1.7.12)

[error] "${{ inputs.apply && 'write' || 'read' }}" is invalid as permission of scope "contents". available values are "read", "write", "none"

(permissions)